### ISSUE TITLE
Release auxiliary keys (Alt, Shift, Ctrl) for custom use

### ISSUE THREAD

**Issue by F-Burning** [tahoma2d/tahoma2d#916]
I think these keys should finally be keyboard shortcuts like in most drawing/animation programs, because in my opinion it feels very limited not being able to use them for work! It would be enough if it were optional in the program's preferences (use them as keyboard shortcuts or as before) It would be great to read more views on it and be able to discuss it.

PD: I take this opportunity to thank you for the enormous work you do week after week to make this possible : ) !

**Comment by artisteacher** [comment]
How would you want to use them?

We can use alt, shift, & ctrl in combination with other keys for custom shortcuts. I don't have any programs that let those keys be set as shortcuts on their own...

Since they are used in so many places - usually for very standardized behaviors, I'd be concerned that using them for custom use would cause problems.

**Comment by F-Burning** [comment]
In Clip Studio Paint for example, you can use them like auxiliary custom keys (Mayus = Shift )

![Screenshot_1](https://user-images.githubusercontent.com/74098734/156142903-d356ee6e-98a0-439e-9412-de83887613cc.jpg)


In krita the same

![6](https://user-images.githubusercontent.com/74098734/156143371-c1c19f83-bba9-41d7-bd27-8bca7d7a019b.jpg)

For example, in Clip studio and Krita you can activate those keys only when using specific tools without compromising the normal operation of the program. If it can be done on these shows I don't see why Tahoma couldn't. Believe me, doing this offers incredible versatility.

My proposal is to do it as something optional that does not compromise the current use of these keys if you don't want to!

**Comment by artisteacher** [comment]
This might be quite complex to get working well. It's not just that a lot of tools use the modifiers in different ways, but they are also used in the Timeline. Which I think was one of the reasons it ended up disabled to begin with.

**Comment by F-Burning** [comment]
Yes, at first it seems somewhat problematic, but Clip Studio Paint and Krita have timelines and 0 problems with this!
If for example you are using the Xsheet in opentoonz by clicking on it, the auxiliary keys should continue to work as usual within that panel, but when drawing in the viewer, they could be the ones you have assigned.

**Comment by F-Burning** [comment]
Don't worry, that's why I propose to make it optional in preferences.

Something like: 

_Use auxiliary keys as keyboard shortcuts_ ✅
